packages/blade-mcp/src/server.ts

```diff
@@ -37,10 +68,129 @@ try {
   });
 
   server.tool(
-    'get_blade_components',
-    `Return the list of Blade Design System components' knowledgebase files required for generating UI code with Blade`,
+    'hi_blade',
+    'Call this when the user says "hi blade", "hey blade" or "namaste blade". Tool that returns how to use blade mcp',
+    {},
+    () => {
+      return {
+        content: [
+          {
+            type: 'text',
+            text: `Print this message as is: ${hiBladeMessage}`,
+          },
+        ],
+      };
+    },
+  );
+
+  server.tool(
+    'create_new_blade_project',
+    'Create a new project using blade. Uses vite, react, and typescript for setup. Call this tool ONLY when you are creating a new project from scratch.',
+    {
+      currentProjectRootDirectory: z
+        .string()
+        .describe("The working root directory of the consumer's project"),
+    },
+    ({ currentProjectRootDirectory }) => {
+      // check if project is not empty
+      if (readdirSync(currentProjectRootDirectory).length > 0) {
+        return {
+          isError: true,
+          content: [
+            {
+              type: 'text',
+              text:
+                'Project is not empty. Call this tool only when creating a new project from scratch.',
+            },
+          ],
+        };
+      }
+
+      // Copy project from __dirname/base-blade-template to consumer's project
+      const baseBladeTemplatePath = resolve(__dirname, '..', 'base-blade-template');
+
+      // copy directory
+      if (existsSync(join(baseBladeTemplatePath, 'node_modules'))) {
+        rmSync(join(baseBladeTemplatePath, 'node_modules'), { recursive: true });
+      }
+
+      cpSync(baseBladeTemplatePath, currentProjectRootDirectory, { recursive: true });
+      // rename gitignore to .gitignore
+      renameSync(
+        join(currentProjectRootDirectory, 'gitignore'),
+        join(currentProjectRootDirectory, '.gitignore'),
+      );
+
+      return {
+        content: [
+          {
+            type: 'text',
+            text: `Project created successfully in ${currentProjectRootDirectory}. 
+            ## AI should run the following commands
+
+            1. Call create_blade_cursor_rules to create the cursor rules.
+            2. Run \`npm install --legacy-peer-deps && npm install @razorpay/blade@latest --legacy-peer-deps\` to install the latest blade.
```

Comment:
why do we need this? what if we just mark "*" in the package.json 🤔. my concern is more around --legacy-peer-deps.

Reply:
`--legacy-peer-deps` we have to do anyways. NPM did that painful strictly match peer dependencies thing change in new versions so it almost always ends up failing. Even with *, we'll still need `--legacy-peer-deps` during installation step